docs/architecture-and-design.md

```diff
@@ -386,6 +389,20 @@ Most [Intents](#intents) are easy to break down into  [Primitives](#primitives),
 
 In order for a robot to move to the desired destination of a `MoveIntent`, the Navigator will use various path-planning algorithms to find a path across the field that does not collide with any robots or violate any restrictions set on the `MoveIntent`. The Navigator then translates this path into a series of `MovePrimitives`, which are sent to the robot sequentially so that it follows the planned path across the field.
 
+### Path Manager
+The `Path Manager` is responsible for generating a set of paths that don't collide. It is given a set of path objectives and path planner and it will generate paths using the given path planner and arbitrate between paths to prevent collisions. One approach is to add extra obstacles to cause the path planner to generate paths that avoid a potential collisions; this is what `VelocityObstaclePathManager` implements. One disadvantage of this approach is that it does not account for when robots will occupy particular positions on the path. (Trajectory planning or use of the Minkowski space are possible ideas to account for time.)
```

Comment:
NIts:
"a set of path objectives and a path planner and it will" -> "a set of path objectives and a path planner, and it will"
"that avoid a potential collisions;" -> "that avoid potential collisions;"